src/psyclone/nemo.py

```diff
@@ -45,7 +45,7 @@
 from psyclone.psyGen import PSy, Invokes, Invoke, Schedule, Node, \
     Loop, Kern, GenerationError, InternalError, colored, IfBlock, IfClause, \
     NameSpaceFactory, SCHEDULE_COLOUR_MAP as _BASE_CMAP
-from fparser.two.utils import walk_ast
+from fparser.two.utils import walk_ast, get_child
```

Comment:
Hmm, don't know why you're getting that error (mind you, pylint frequently gets confused about things). The branch must point to the correct fparser because the tests pass on Travis. Have you tried doing a `git submodule update`?

Reply:
I've had this problem before and in my case it was to do with pointing to the wrong version of fparser. I uninstalled fparser then installed pointing at the submodule in psyclone and it worked.